modules/apigee_asyncapi_doc/apigee_asyncapi_doc.libraries.yml

```diff
@@ -0,0 +1,24 @@
+# Third-party library (CDN).
+reactjs:
+  remote: https://reactjs.org
+  version: 17.0.2
+  license:
+    name: MIT
+    url: https://github.com/facebook/react/blob/main/LICENSE
+    gpl-compatible: true
+  js:
+    https://unpkg.com/react@17.0.2/umd/react.production.min.js: {type: external, minified: true}
```

Comment:
I had a conversation with Marsh and we want to minimize changes to this module whenever AsyncAPI library releases a security patch.

The only way to do this is to point to the latest version and remove the dependency of specific version from our module.

Can we change the URL of these assets to autofetch the latest version.

Reply:
Done